### PR TITLE
Make bugbear slightly more polite

### PR DESCRIPTION
**Pull request by sam-w**
Bugbear is great at catching bugs in our code, but can come across a bit rude when commenting on pull requests 😂  Obviously it's a robot, but we like to encourage polite discourse

### CODE REVIEW

File: bugbear.py [hunk 1]
```diff
@@ -1213,7 +1213,8 @@ def visit_Lambda(self, node):
 )
 B018 = Error(
     message=(
-        "B018 Found useless expression. Either assign it to a variable or remove it."
+        "B018 Found useless expression. Consider either assigning it to a"
```
Comment (cooperlees) [inline bugbear.py:1216]:
```suggestion
        "B018 Found useless expression. Consider either assigning it to a "
```
Whitespace

File: bugbear.py [hunk 2]
```diff
@@ -1192,8 +1192,8 @@ def visit_Lambda(self, node):
 }
 B015 = Error(
     message=(
-        "B015 Pointless comparison. This comparison does nothing but waste "
-        "CPU instructions. Either prepend `assert` or remove it."
```
Comment (cooperlees) [inline bugbear.py:1196]: I personally feel the blunt nature of the message makes it clearer if what the check looks for.
The new message will make people think into it more and possibly over complicate their thoughts. But open to others thoughts …

**Review by cooperlees — APPROVED** [review]
Seems fair. I’d like others thoughts if anyone comes by …

File: bugbear.py (hunk 2 above)
Comment (sam-w) [reply bugbear.py:1196]: I definitely see the tradeoff here, and agree that the cause of the error should be the first thing the user sees. The aim with the change of wording was to retain that, while softening the order to a suggestion.

**Review by cooperlees — APPROVED** [review]
Thanks! Sorry about being slow. I’m on holidays at the moment.

**Comment by sam-w** [comment]
@cooperlees no probs! Thanks for accepting the change, hope your holidays were excellent

**Pull request merged**